src/API/src/allDataFileBuilder.service.test.ts

```diff
@@ -0,0 +1,77 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { OccurrenceService } from 'src/db/occurrence/occurrence.service';
+import { MockType, repositoryMockFactory } from 'src/mocks';
+import { Logger } from '@nestjs/common';
+import { ExportService } from './export/export.service';
+import { AllDataFileBuilder } from './allDataFileBuilder.service';
+import { getRepositoryToken } from '@nestjs/typeorm';
+import { Occurrence } from 'src/db/occurrence/entities/occurrence.entity';
+import { handleLastIngestLock } from 'src/ingest/utils/triggerCsvRebuild';
+const fs = require('fs');
```

Comment:
As you say, this needs to be fixed. Can you not use `import * as fs from 'fs';`?